docs/user/meta_multi_task_rl_exp.md

```diff
@@ -0,0 +1,304 @@
+            hidden_sizes=(20, 10),
+            std_share_network=True,
+            init_std=2.0,
+            output_nonlinearity=tf.nn.tanh,
+            min_std=embedding_min_std,
+        )
+
+        policy = GaussianMLPTaskEmbeddingPolicy(
+            name='policy',
+            env_spec=env.spec,
+            encoder=task_encoder,
+            hidden_sizes=(32, 16),
+            std_share_network=True,
+            max_std=policy_max_std,
+            init_std=policy_init_std,
+            min_std=policy_min_std,
+        )
+
+        baseline = LinearMultiFeatureBaseline(
+            env_spec=env.spec, features=['observations', 'tasks', 'latents'])
+
+        algo = TEPPO(env_spec=env.spec,
+                     policy=policy,
+                     baseline=baseline,
+                     inference=inference,
+                     max_path_length=max_path_length,
+                     discount=0.99,
+                     lr_clip_range=0.2,
+                     policy_ent_coeff=policy_ent_coeff,
+                     encoder_ent_coeff=encoder_ent_coeff,
+                     inference_ce_coeff=inference_ce_coeff,
+                     use_softplus_entropy=True,
+                     optimizer_args=dict(
+                         batch_size=32,
+                         max_epochs=10,
+                         learning_rate=1e-3,
+                     ),
+                     inference_optimizer_args=dict(
+                         batch_size=32,
+                         max_epochs=10,
+                     ),
+                     center_adv=True,
+                     stop_ce_gradient=True)
+
+        runner.setup(algo,
+                     env,
+                     sampler_cls=LocalSampler,
+                     sampler_args=None,
+                     worker_class=TaskEmbeddingWorker)
+        runner.train(n_epochs=n_epochs, batch_size=batch_size, plot=False)
+
+
+te_ppo_mt50()
+
+```
+
+In this example, to sample tasks from `MT50` environments in a round robin fashion, the call to `MultiEnvWrapper` becomes `MultiEnvWrapper(envs , sample_strategy=round_robin_strategy, mode='del-onehot')`.
+
+## Garage's meta-/ multi-task RL benchmark
+
+Garage benchmarks the following meta-/ multi RL experiements:
+
+```eval_rst
+| Algorithm     | Observation | Action   | Environment Set                                                            |
+|---------------|-------------|----------|----------------------------------------------------------------------------|
+| Meta-RL       | Non-Pixel   | Discrete | 'ML1-push-v1' ,  'ML1-reach-v1' ,  'ML1-pick-place-v1' ,  'ML10' ,  'ML45' |
+| Multi-Task RL | Non-Pixel   | Discrete | 'ML1-push-v1' ,  'ML1-reach-v1' ,  'ML1-pick-place-v1' ,  'MT10' ,  'MT50' |
+|               |             |          |                                                                            |
+```
+
+See `docs/benchmarking.md` for a more detailed explaination on garage's benchmarking.
```

Comment:
I think this should link to the document being referred. A file system path in documentation is not useful.

Reply:
You're right. Added the doc link.